server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/step/LoadReportAnalysisMetadataHolderStep.java

```diff
@@ -86,12 +87,19 @@ public void execute() {
     ScannerReport.Metadata reportMetadata = reportReader.readMetadata();
 
     loadMetadata(reportMetadata);
+    loadOrganizationsEnabled();
     Organization organization = loadOrganization(reportMetadata);
     loadProject(reportMetadata, organization);
     loadQualityProfiles(reportMetadata, organization);
     branchLoader.load(reportMetadata);
   }
 
+  private void loadOrganizationsEnabled() {
+    try (DbSession dbSession = dbClient.openSession(false)) {
```

Comment:
there is no reason to open a DB connection twice in a row. Even with a connection pool, that will uselessly consume extra resources.
you should refactor and use the same DB session in `loadOrganizationsEnabled` and `loadMetadata`